src/components/breadcrumbs/breadcrumbs.stories.mdx

```diff
@@ -0,0 +1,87 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+import { Breadcrumbs, Crumb } from "./index";
+import * as stories from "./breadcrumbs.stories.tsx";
+import TranslationKeysTable from "../../../.storybook/utils/translation-keys-table";
+
+<Meta title="Breadcrumbs" parameters={{ info: { disable: true } }} />
+
+# Breadcrumbs
+
+  <a
+  target="_blank"
+  href="https://zeroheight.com/2ccf2b601/p/32c1b6-breadcrumbs/b/87b2c7"
+  style={{ color: "#007E45", fontWeight: "bold", textDecoration: "underline" }}
+  >
+    Product Design System component
+  </a>
+
+Breadcrumbs are a secondary navigation aid that helps users easily understand their location.
+
+## Contents
+
+- [Quick Start](#quick-start)
+- [Examples](#examples)
+- [Props](#props)
+
+## Quick Start
+
+To use Breadcrumbs component you need to import `Breadcrumbs` and pass an array of `Crumb`s.
+
+```javascript
+import { Breadcrumbs, Crumb } from "carbon-react/lib/components/breadcrumbs";
+```
+
+## Examples
+
+### Breadcrumbs
+
+<Canvas>
+  <Story name="default" story={stories.DefaultBreadcrumbs} />
+</Canvas>
+
+## Props
+
+### Breadcrumbs
+
+<ArgsTable
+   of={Breadcrumbs}
+ />
+
+### Crumb
+
+<ArgsTable of={Crumb} 
+  exclude={[
+     "className",
+     "onKeyDown",
```

Comment:
I think the event handlers should be displayed here so `onKeyDown`, `onClick` and `onMouseDown` shouldn't be excluded. IMO the only props excluded should be the ones omitted from the `CrumbProps` interface

`Omit<LinkProps, "toolTipMessage" | "tooltipPosition" | "iconType" | "iconAlign" | "isSkipLink" | "isDarkBackground" | "ariaLabel" | "className">`